sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_link_async.py

```diff
@@ -115,11 +115,16 @@ def _check_if_closed(self):
             try:
                 raise self._error
             except TypeError:
-                raise AMQPConnectionError(condition=ErrorCondition.InternalError, description="Link already closed.")
+                raise AMQPConnectionError(condition=ErrorCondition.InternalError,
+                                          description="Link already closed.") from None
 
     async def _set_state(self, new_state):
         # type: (LinkState) -> None
-        """Update the session state."""
+        """Update the session state.
+        :param ~pyamqp.constants.SessionState new_state: The new state.
```

Comment:
This is updating the link state
```suggestion
        :param ~pyamqp.constants.LinkState new_state: The new state.
```